File: cdf-785A.cpp
#include<iostream>
using namespace std;
int main(){
    int n;
    cin>>n;
    string a[]={"Tetrahedron","Cube","Octahedron","Dodecahedron","Icosahedron"};
    int sum=0;

   for(int i=0;i<n;i++){
         string s;
         cin>>s;
        if(s==a[0])
           sum+=4;
          else if(s==a[1])
           sum+=6;
          else if(s==a[2])
             sum+=8;
           else if(s==a[3])
             sum+=12;
           else if(s==a[4])
                sum+=20;
   }
   cout<<sum<<endl;
    return 0;
}

File: cdf-785A.py
n = int(input())


faces = {
    "Tetrahedron": 4,
    "Cube": 6,
    "Octahedron": 8,
    "Dodecahedron": 12,
    "Icosahedron": 20
}

ans = 0
for i in range(n):
    
    name = input().strip()
  
    ans += faces[name]

print(ans)